### ISSUE TITLE
Missing @cdktf/provider-vault

### ISSUE THREAD

**Issue by OlivierPT** [hashicorp/terraform-cdk#1897]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I am missing a native CDK TF provider for VAULT (@cdktf/provider-vault). 
Could it be added to all the other standard native provider.

If this is possible to contribute to make it happen, I will be happy to to it. I would just need the link to know what it the process to follow.

### References

None.

**Comment by DanielMSchmidt** [comment]
Created https://github.com/hashicorp/cdktf-repository-manager/pull/75 to add it

**Comment by OlivierPT** [comment]
Thanks a lot @DanielMSchmidt !
At the moment, we are using `cdktf.json` solution indeed. But we build some lib based on CDKTF with dependencies and it make it easier to manage to have them all as native CDK dependencies... :)

**Comment by DanielMSchmidt** [comment]
Yeah that is right :) Here is the provider: https://github.com/hashicorp/cdktf-provider-vault

It's easy to assume we only support a subset of providers, that's why I wanted to mention that we support all of them, some are just not prebuilt yet :)
